src/core/connection.c

```diff
@@ -1129,6 +1143,7 @@ QuicConnTimerSetEx(
     )
 {
     const uint64_t NewExpirationTime = TimeNow + Delay;
+    uint64_t NewEarliestExpirationTime;
```

Comment:
Since there doesn't seem to be a `goto` between here and when the variable is written, just remove this line and add the `uint64_t` below, where it is set.